homeassistant/components/nuki/lock.py

```diff
@@ -26,29 +27,32 @@
 NUKI_DATA = 'nuki'
 
 SERVICE_LOCK_N_GO = 'nuki_lock_n_go'
```

Comment:
Since we're changing domain of the service to 'nuki', remove `nuki_` from the service name.

Reply:
Done